packages/console/src/SerialConsole/__snapshots__/SerialConsole.test.js.snap

```diff
@@ -33,7 +33,7 @@ exports[`Render SerialConsole in the CONNECTED state 1`] = `
 
 exports[`SerialConsole in the CONNECTED state 1`] = `
 <div
-  className=""
+  className="serial-console-pf "
```

Comment:
can we add a tests for adding the optional topClassName and autoFit parameters?

Reply:
Will add tests